Changed to allow versions greater than 9.73.2

Calerid opened the pull request.

Calerid committed:
Implimented TWS Server version check

Calerid committed:
Merge branch 'michaels-branch' of https://github.com/tranceitionalMynd/spy-reaper into michaels-branch

Calerid committed:
Edited server version check: Error Message

Made server version error string literal (server_ver_req).

tranceitionalMynd commented:
Before merging this PR, I need to think about how to fix duplicate commits that got created when I rebase and merged your last PR.

Calerid committed:
Implemented Basic Stock Historical Data 

Basic script to call data from Alpha Vantage. Calls data to file "history.json".

Calerid committed:
Renamed to data.py / Deleted SubDirectory / Deleted extra

Renamed historical_data.py to data.py, deleted example sub directory, deleted the example of the historical.json file in sub directory.

Calerid commented:
Maybe one day we will revisit this again.
